src/en/config-LXD.md

```diff
@@ -54,53 +59,56 @@ On Wily, add this:
 
 ```bash
 sudo add-apt-repository -y ppa:ubuntu-lxc/lxd-stable
-```
-
-Proceed with the rest:
-
-```bash
-sudo apt-add-repository -y ppa:juju/devel
 sudo apt-get update
-sudo apt-get install juju-local lxd
+sudo apt-get install lxd
 ```
 
-Now either log out and log in again to get your group membership refreshed, or use:
+Now either log out and log in again to get your group membership refreshed, or
+use:
 
 ```bash
 newgrp lxd
 ```
 
+Xenial is shipped with LXD pre-installed.
```

Comment:
Since I already mentioned the entire label I felt I didn't need to do it again. I'll add it all if you want.